src/settings/networkremotesettingspage.cpp

```diff
@@ -0,0 +1,99 @@
+#include <QStyle>
+#include "core/iconloader.h"
+#include "core/logging.h"
+#include "networkremote/networkremote.h"
+#include "settings/settingsdialog.h"
+#include "settings/networkremotesettingspage.h"
+#include "ui_networkremotesettingspage.h"
+
+using namespace Qt::Literals::StringLiterals;
+
+NetworkRemoteSettingsPage::NetworkRemoteSettingsPage(SettingsDialog *dialog, QWidget *parent) :
+  SettingsPage(dialog,parent),
+  ui_(new Ui_NetworkRemoteSettingsPage),
+  settings_(new NetworkRemoteSettings)
+{
+  ui_->setupUi(this);
+  const int iconSize = style()->pixelMetric(QStyle::PM_TabBarIconSize);
+  setWindowIcon(IconLoader::Load(QStringLiteral("network-remote"), true, 0,iconSize));
+  QObject::connect(ui_->useRemoteClient,&QAbstractButton::clicked, this, &NetworkRemoteSettingsPage::RemoteButtonClicked);
+  QObject::connect(ui_->localConnectionsOnly, &QAbstractButton::clicked, this, &NetworkRemoteSettingsPage::LocalConnectButtonClicked);
+  QObject::connect(ui_->portSelected, &QAbstractSpinBox::editingFinished, this, &NetworkRemoteSettingsPage::PortChanged);
+}
+
+NetworkRemoteSettingsPage::~NetworkRemoteSettingsPage()
+{
+  delete ui_;
+}
+
+void NetworkRemoteSettingsPage::Load()
+{
+  ui_->portSelected->setRange(5050, 65535);
+  ui_->ip_address->setText(QStringLiteral("0.0.0.0"));
```

Comment:
You included Qt::Literals::StringLiterals but you forgot to actually use it.

Reply:
Removed